src/instance/instanceinterfaces.ts

```diff
@@ -13,6 +14,8 @@ export enum State {
   Canceled = 3,
 }
 
+export const StateSchema = Joi.number().max(3).min(1);
```

Comment:
Same here as for the other enum, I think it would be good to have a simple test to get notified if the enum changes if one forgets to adjust these values.